demos/Shrine/Podfile.lock

```diff
@@ -45,9 +45,9 @@ DEPENDENCIES:
 
 EXTERNAL SOURCES:
   MaterialComponents:
-    :path: "../../"
```

Comment:
sorry about this but can you revert the quotes?
https://github.com/material-components/material-components-ios/issues/805